### ISSUE TITLE
class History not returning all metrics.

### ISSUE THREAD

**Issue by michellebarbosa** [adap/flower#687]
Hi! Im trying to retrieve the values of accuracie_distributed, losses_centralised and accuracies_centralised but when I run the clients on my machine it returns an empty list. What could be the reason for this? TIA.

`INFO flower 2021-03-26 08:50:15,330 | app.py:109 | app_fit: losses_distributed [(1, 0.07582958787679672), (2, 0.06492161005735397), (3, 0.07073713093996048)] 
INFO flower 2021-03-26 08:50:15,330 | app.py:110 | app_fit: accuracies_distributed [] 
INFO flower 2021-03-26 08:50:15,330 | app.py:111 | app_fit: losses_centralized [] 
INFO flower 2021-03-26 08:50:15,331 | app.py:112 | app_fit: accuracies_centralized []`

**Comment by danieljanes** [comment]
Hi @michellebarbosa , thanks for reaching out. Are you using centralized or distributed evaluation?

**Comment by michellebarbosa** [comment]
I've worked on multiple multple connected to the same network, also worked on training multiple clients on one computer. Could you provide a little clarity on what a distributed evaluation would be? I haven't come across any mention on the documentation or Github repo. Thanks!

**Comment by danieljanes** [comment]
Right, the documentation for evaluation was something we wanted to do for a while. Your issue motived us to write it up, here's a first draft: https://github.com/adap/flower/pull/691/files

Let me know what you think!
